skins/tao/editor.css

```diff
@@ -67,6 +67,8 @@ Usage:
     box-sizing: content-box;
     position: static;
     transition: none;
+	position: relative;
+	z-index: 0;
```

Comment:
is it actually a tab I see there? By convention we must use spaces instead of tabs.

Reply:
Check again, maybe issue when copy/paste from browser